src/Server.ts

```diff
@@ -0,0 +1,32 @@
+import IConfig from './config/IConfig';
+import * as express from 'express';
+
+export default class Server {
+    app: express.Application;
+    constructor( private config: IConfig) {
+        this.app = express();
+    }
+    bootstrap = () => {
+        this.setupRoutes();
+        return this;
+    }
+
+    run = () => {
+        const{app, config: { port , env } } = this;
+        this.app.listen(port, (err) => {
+            if (err) {
+                throw err;
+            }
+        });
+       console.log('App is running successfully at ', port, env);
+       return this;
+    }
+
+    setupRoutes = () => {
+        this.app.get('/health-check', ( req, res) => {
```

Comment:
use app.use not get, also go through the difference between using use and get here